dfastmi/gui/dialog_view_model.py

```diff
@@ -286,8 +311,8 @@ def _update_qvalues(self) -> None:
             if self.current_reach.auto_time:
                 _, time_mi = ConfigurationInitializer.set_times(
                     self._current_reach.hydro_q,
```

Comment:
Why using field instead of property?

Reply:
@ralph-peelen-deltares, Would you be able to discuss this shortly with Bert and explain it further.

Reply:
I noticed an inconsistency between using the "_current_reach" field and in the line above and the "current_reach" property in the lines below. I decided to make these consistent. Since it's inside the objects own code, I thought that using the fields would be fine. If the property would be doing something special the we would be doing it three times ... or we should first get the current_reach and then the hydro_q, qfit and qstagnant properties of that item.